### ISSUE TITLE
Errors appearing in only in dash of subscriber users

### ISSUE THREAD

**Issue by DWScode** [pojome/elementor#6936]
I've got some unusual errors appearing only in the subscriber level dashboard. (Errors do not appear on admin dash)

It appears on both the live site and offline local dev site.

The errors appear with the only plugin running Elementor v2.4.5 and with the Twenty Nineteen theme running. So it doesn't appear to be a custom theme / plugin conflict. I even went so far as to remove the Elementor widget on the subscriber dash to see if that was causing the problem but nope - errors still there.

The following errors appear in the main dash area:
```
Warning: Invalid argument supplied for foreach() in _[the site folder]_/wp-admin/includes/plugin.php on line 1566

Warning: Cannot modify header information - headers already sent by (output started at _[the site folder]_/wp-admin/includes/plugin.php:1566) in _[the site folder]_/wp-admin/includes/misc.php on line 1144

Warning: Cannot modify header information - headers already sent by (output started at _[the site folder]_/wp-admin/includes/plugin.php:1566) in _[the site folder]_/wp-includes/option.php on line 919

Warning: Cannot modify header information - headers already sent by (output started at _[the site folder]_/wp-admin/includes/plugin.php:1566) in _[the site folder]_/wp-includes/option.php on line 920

Warning: Invalid argument supplied for foreach() in _[the site folder]_/wp-admin/includes/plugin.php on line 1566
```
That last error also appears in the admin menu bar in the left column of the subscriber dash (above both the "dash" and then again above the "profile" menu links):

Below are my live server specs:
<details>
<summary>System Info</summary>

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 84.1, Revision b308619 v5.6.41-84.1
	PHP Version: 7.1.14
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.0.3
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Permalink Structure: /%category%/%postname%/
	Language: en-US
	Timezone: America/Chicago
	Debug Mode: Inactive

== Theme ==
	Name: Colormag Child Theme
	Version: 1.0
	Author: ThemeGrill
	Child Theme: Yes
	Parent Theme Name: ColorMag
	Parent Theme Version: 1.3.5
	Parent Theme Author: ThemeGrill

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36

== Active Plugins ==
	Crop Thumbnails
		Version: 1.2.3
		Author: Volkmar Kantor

	Duplicator
		Version: 1.3.4
		Author: Snap Creek

	Elementor
		Version: 2.4.5
		Author: Elementor.com

	Everest Forms
		Version: 1.4.4
		Author: WPEverest

	FSM Custom Featured Image Caption
		Version: 1.17
		Author: Fesomia

	Google Analytics Dashboard for WP (GADWP)
		Version: 5.3.7
		Author: ExactMetrics

	Hummingbird
		Version: 1.9.3
		Author: WPMU DEV

	iThemes Security
		Version: 7.2.0
		Author: iThemes

	LoginPress - Customizing the WordPress Login
		Version: 1.1.19
		Author: WPBrigade

	Nextend Social Login
		Version: 3.0.17
		Author: Nextendweb

	Pretty Links
		Version: 2.1.7
		Author: Blair Williams

	Redirection
		Version: 3.7.2
		Author: John Godley

	Responsive Lightbox & Gallery
		Version: 2.1.0
		Author: dFactory

	Smush
		Version: 3.0.2
		Author: WPMU DEV

	ThirstyAffiliates
		Version: 3.7
		Author: Rymera Web Co

	TinyMCE Advanced
		Version: 5.0.1
		Author: Andrew Ozz

	UpdraftPlus - Backup/Restore
		Version: 1.16.5
		Author: UpdraftPlus.Com, DavidAnderson

	Wordfence Security
		Version: 7.1.20
		Author: Wordfence

	WP-Optimize
		Version: 2.2.11
		Author: David Anderson, Ruhani Rabin, Team Updraft

	wpDiscuz
		Version: 5.3.1
		Author: gVectors Team (A. Chakhoyan, G. Zakaryan, H. Martirosyan)

	Yoast SEO
		Version: 9.5
		Author: Team Yoast

== Log ==
	: 
Log: showing 3 of 32019-01-21 20:55:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.1',
  'to' => '2.4.3',
)]
2019-01-24 13:08:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.3',
  'to' => '2.4.4',
)]
2019-01-31 15:19:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
</details>

**Comment by VarvatosVex** [comment]
I am experiencing this error as well. In my case while running the Hueman theme.
It only appears in subscriber level dashboard (all above levels are spared), and shows up at the top of the site, as well as in the sidebar. It affects both the dashboard page and the profile page.

Some screenshots for clarity:
![image](https://user-images.githubusercontent.com/20911914/52161236-7205bb80-26c2-11e9-87de-7453cb3a60c2.png)
![image](https://user-images.githubusercontent.com/20911914/52161237-7cc05080-26c2-11e9-895c-51a38a38241e.png)

It's worth mentioning that the issue has persisted through every single version from 2.40 to 2.4.5, and the only fix is to downgrade back to 2,3,8.

I can't get a full list of my plugins right now, but I can verify that the only two plugins me and DWScode share is the "Google Analytics Dashboard for WP" and the "Yoast SEO" plugins (and of course Elementor). Same versions.

**Comment by shilo-ey** [comment]
Hi @DWScode & @VarvatosVex 

Thanks for your feedback.

We took care of this issue and will fix it in the next release.

**Comment by shilo-ey** [comment]
Hi @DWScode @VarvatosVex 

This has been fixed in Elementor 2.4.6.

Feel free to update!

thanks.

**Comment by DWScode** [comment]
I can verify the bug has been fixed! Thanks!
